src/tracer.js

```diff
@@ -80,6 +80,9 @@ export default class Tracer {
             } else {
                 fields.operationName = nameOrFields;
             }
+            if (fields.parent) {
+                fields.parent = fields.parent._imp;
```

Comment:
Internal implementation have no idea where interface store implementations unless dig into the source code. And interface may forbid outside world to access it's internal implementation!

Take a look at https://github.com/opentracing/opentracing-javascript/blob/master/src%2Ftracer.js#L138, at least we should be consistency, isn't it?

Another thought about modify arguments, there are simple way to fix this

```js
startSpan(operationName, fields) {
  if (API_CONFORMANCE_CHECKS) { ... }
  return new Span(this._imp.startSpan({
    operationName: operationName,
    parent: fields.parent._imp,
    // other fields
  }));
}
```